src/java/fr/paris/lutece/plugins/html/business/portlet/BaseHtmlPortletDAO.java

```diff
@@ -95,13 +99,15 @@ public void delete( int nPortletId )
     public void store( Portlet portlet )
     {
         IHtmlPortlet p = (IHtmlPortlet) portlet;
-        DAOUtil daoUtil = new DAOUtil( SQL_QUERY_UPDATE );
-        daoUtil.setInt( 1, p.getId( ) );
-        daoUtil.setString( 2, p.getHtml( ) );
-        daoUtil.setInt( 3, p.getId( ) );
+        try ( DAOUtil daoUtil = new DAOUtil( SQL_QUERY_UPDATE ) )
+        {
+        	daoUtil.setInt( 1, p.getId( ) );
+            daoUtil.setString( 2, p.getHtml( ) );
+            daoUtil.setInt( 3, p.getId( ) );
 
-        daoUtil.executeUpdate( );
-        daoUtil.free( );
+            daoUtil.executeUpdate( );
+            daoUtil.free( );
```

Comment:
daoUtil.free( ): To be removed since the stream is already closed at the end of the try block.